src/services/api/AuthService.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosResponse } from "axios";
+import { api } from "../../http";
+import { authResponse, IOrder, IProduct } from "./types";
```

Comment:
Types should be called from upper case.
```suggestion
import { AuthResponse, IOrder, IProduct } from "./types";
```